yarn.lock

```diff
@@ -974,10 +974,8 @@
     mkdirp "^1.0.4"
     rimraf "^3.0.2"
 
-"@powerhousedao/design-system@^1.0.0-alpha.12":
+"@powerhousedao/design-system@file:.yalc/@powerhousedao/design-system":
```

Comment:
Is this because of https://github.com/powerhouse-inc/design-system/pull/82?
It's ready to merge and release.

Reply:
yes, added the correct dependency here: aefd83be7c1b0ab035eb0504970fd8ae714f68ab